lib/content-services/document-list/data/share-datatable-adapter.ts

```diff
@@ -118,8 +118,10 @@ export class ShareDataTableAdapter implements DataTableAdapter {
             }
 
             if (node.entry.isFile) {
-                if (this.thumbnails) {
+                if (this.thumbnails && node.entry.nodeType != "app:filelink") {
                     return this.documentListService.getDocumentThumbnailUrl(node);
+                } else if (this.thumbnails && "cm:destination" in node.entry.properties) {
```

Comment:
It is missing the final fallback when `node.entry.isFile` and properties have no "app:fileLink" and no "cm:destination"

Reply:
Yes. But what would the fallback be?

Reply:
I don't get the question. The code execution will go to the `return this.documentListService.getDefaultMimeTypeIcon();` if you check the whole code.